http/src/routing.rs

```diff
@@ -1437,11 +1439,22 @@ impl Route {
 
                 (Method::Get, Path::UsersIdGuilds, path.into())
             }
-            Self::GetInvite { code, with_counts } => (
-                Method::Get,
-                Path::InvitesCode,
-                format!("invites/{}?with-counts={}", code, with_counts).into(),
-            ),
+            Self::GetInvite {
+                code,
+                with_counts,
+                with_expiration,
```

Comment:
This is a breaking change, can we do a `GetInviteWithExpiration` route? Then we can delete it in `next` and merge it in with `GetInvite`.